File: CPP/High_Voltage/GlobalVariables.cpp
#include "GlobalVariables.h"

namespace Shared_Vars{
    extern int nInterAlive{};
    extern int nInterKilled{};
    extern int nTotalinterceptorsKilled{};
    extern std::string rMaze[iMazeRows][iMazeCols]{};
    extern HANDLE hStdout{};
    extern HANDLE hStdin{};
    extern CONSOLE_SCREEN_BUFFER_INFO csbiInfo{};
    extern cPlayer cActivePlayer{};
    extern std::vector<cInterceptor> interceptorCollection{};
}

File: HTML/High_Voltage_JS/GlobalVariables.js


const sOpenAreaSymbol = " ";
const sPlayerSymbol = "*" ;
const sDeadPlayerSymbol ="#" ;
const sInterceptorSymbol = "+" ;
const iNumberOfinterceptors = 15 ;
const sHighVoltageSymbol = "X" ;
const iMaxHighVoltageBarriersPerRow = 15 ;

const iMazeRows = 25 ;
const iMazeCols = 30 ;


let nInterAlive =iNumberOfinterceptors;
let nInterKilled=0;
let nTotalinterceptorsKilled=0;
let rMaze = [sHighVoltageSymbol];
let cActivePlayer = new cPlayer;    
let interceptorCollection = [sInterceptorSymbol];
